templates/autonomous-car.html

```diff
@@ -0,0 +1,35 @@
+<!doctype html>
+<html>
+<head>
+    <title>Autonomous Car Assignment</title>
+</head>
+<body>
+    <h3>Autonomous Car Assignment</h3>
+    <div>Build an application / tool in React or Python with the following criteria:</div>
+    <ul>
+        <li>Fetches a car route from this API server</li>
+        <li>Determines if the route was a success or a failure</li>
+        <li>If the route results in a failure, determine the cause as well as the step and position where the failure occurred</li>
```

Comment:
The instructions need more detail. There's no indication that there's only three lanes. Can you update the instructions from here: https://github.com/Points/developer-assignment-instructions/tree/master/autonomous-car